### PR TITLE
Update Cray docs for quicker quick-start instructions

### PR DESCRIPTION
**Pull request by ben-albrecht**
This is coupled to the planned removal of the Cray `README.rst`. This change ensures that the contents of `cray.rst` subsumes the quick-start instructions of the Cray `README.rst`.

### CODE REVIEW

**Comment by ben-albrecht** [comment]
@gbtitus Would be interested in your feedback.

File: doc/release/platforms/cray.rst [hunk 1]
```diff
@@ -17,19 +17,36 @@ Getting Started with Chapel on Cray X-Series Systems
...
+
+     chpl -o hello6-taskpar-dist $CHPL_HOME/examples/hello6-taskpar-dist.chpl
+
+
+3) Execute the resulting executable (on four locales)::
+
+     ./hello6-taskpar-dist -nl 4
+
+
+This is all that is necessary for most Chapel installations on Cray X-Series
```
Comment (gbtitus) [inline doc/release/platforms/cray.rst]: I think the "most Chapel installations" here might be a little too strong.  This gets you everything default, but there are lots of reasons a user might want to change at least something in the configuration.  Maybe rephrase as "This may be all you need to do to use Chapel on a Cray X-Series system."

File: doc/release/platforms/cray.rst [hunk 2]
```diff
@@ -17,19 +17,36 @@ Getting Started with Chapel on Cray X-Series Systems
...
+systems.  If the installation setup by your system administratordeviates from
```
Comment (gbtitus) [inline doc/release/platforms/cray.rst]: missing space before "deviates"

Comment (ben-albrecht) [inline doc/release/platforms/cray.rst]: whoops!

File: doc/release/platforms/cray.rst (hunk 1 above)
Comment (ben-albrecht) [inline doc/release/platforms/cray.rst]: can do!

File: doc/release/platforms/cray.rst [hunk 3]
```diff
@@ -17,19 +17,36 @@ Getting Started with Chapel on Cray X-Series Systems
...
+the default settings, or you are interested in additional configuration
```
Comment (gbtitus) [inline doc/release/platforms/cray.rst]: Maybe "other configuration settings" rather than "additional configuration options".  It's more that you're setting some of them differently, than that there are more of them.

File: doc/release/platforms/cray.rst [hunk 4]
```diff
@@ -17,19 +17,36 @@ Getting Started with Chapel on Cray X-Series Systems
...
+options, see `Using Chapel on a Cray System`_ below.  If instead you wish to
+build Chapel from source, continue on to
+`Building Chapel for a Cray System from Source`_ just below.
 
 For information on obtaining and installing the Chapel module please
 contact your system administrator.
 
 
+
```
Comment (gbtitus) [inline doc/release/platforms/cray.rst]: Did you want 3 blank lines here?

**Comment by gbtitus** [comment]
This looks good modulo those few line comments!
